README.md

```diff
@@ -161,25 +161,60 @@ Harness the power of the [Vuetify community](https://community.vuetifyjs.com) an
 
 ###### Browser Support
 
-Vuetify supports all **modern browsers**, including IE11 and Safari 9+ (using polyfills). From mobile to laptop to desktop, you can rest assured that your application will work as expected. Interested in the bleeding edge? Try the vue-cli Webpack SSR (Server side rendered) template and build websites optimized for SEO.
+Vuetify supports all **modern browsers**, including IE11 and Safari 9+ (using polyfills). From mobile📱 to laptop 💻 to desktop 🖥, you can rest assured that your application will work as expected.
+
+Interested in the bleeding edge? Try the vue-cli Webpack SSR (Server side rendered) template and build websites optimized for SEO.
+
+For more information about IE11 and Safari 9+ polyfills, visit our [Quick Start Guide](https://vuetifyjs.com/en/getting-started/quick-start#ie11-safari-9-support)
 
 ### Documentation
 
-Link to <a href="https://vuetifyjs.com?ref=github" target="_blank">Vuetify documentation</a>.
+To view our docs 📑, get support 🤯 and the store 🛒 visit us at <a href="https://vuetifyjs.com?ref=github" target="_blank">vuetifyjs.com</a>.
+
+### Getting Started
+
+To get started with Vuetify, you can follow one of these simple set-up instructions.
+
+#### One Click Quick-start
 
-### One Click Quick-start
+Looking to dive right in with zero setup and downtime? Check out our [CodePen.io One Minute Quickstart](https://codepen.io/johnjleider/pen/jYZwVZ)
```

Comment:
Would change to `template.vuetifyjs.com` in case actual codepen template link actually changes

Reply:
Changed~